src/lib/viewers/media/MP4.scss

```diff
@@ -34,4 +34,12 @@
             opacity: 1;
         }
     }
+
+    .bp-VideoControlsRoot {
+        position: absolute;
+        right: 0;
+        bottom: -1px;
```

Comment:
What is this `-1px` positioning for?

Reply:
I've noticed there can be a 1px of video showing through below the controls, so this allows it to be covered up by the semi-transparent controls background.